<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SMHoleMag</class>
 <widget class="QWidget" name="SMHoleMag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="HelpButton" name="b_help">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../Resources/pyleecan.qrc">:/images/images/icon/help_16.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="out_hole_pitch">
       <property name="text">
        <string>Slot pitch = 2*Pi / Zs = </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="b_add">
       <property name="text">
        <string>Add new holes</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QTabWidget" name="tab_hole">
     <property name="minimumSize">
      <size>
       <width>770</width>
       <height>500</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="tabsClosable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="b_plot">
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b_previous">
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b_next">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HelpButton</class>
   <extends>QLabel</extends>
   <header>pyleecan/GUI/Tools/HelpButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Resources/pyleecan.qrc"/>
 </resources>
 <connections/>
</ui>
